deployment/variables.tf

```diff
@@ -52,23 +46,10 @@ variable "environment" {
 variable "container_image" {
   type        = string
   description = "Deployed container name"
-  default     = "ghcr.io/trento-project/telemetry"
+  default     = "ghcr.io/trento-project/telemetry:rolling"
 }
 
-variable "container_tag" {
+variable "lb_certificate_arn" {
```

Comment:
Should we add the `dns_zone` variable even here? In the main `variables.tf`.
Otherwise, it is not really a variable, since we would need to change the code to use a different value

Reply:
yes, definitely! I will add both dns variables (i just added a second one for the CNAME record).